Calibration/EcalAlCaRecoProducers/plugins/SelectedElectronFEDListProducer.cc

```diff
@@ -309,31 +296,26 @@ void SelectedElectronFEDListProducer<TEle, TCand>::beginJob() {
 template <typename TEle, typename TCand>
 void SelectedElectronFEDListProducer<TEle, TCand>::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
   // get the hcal electronics map
-  edm::ESHandle<HcalDbService> pSetup;
-  iSetup.get<HcalDbRecord>().get(pSetup);
+  const auto pSetup = iSetup.getHandle(hcalDbToken_);
   HcalReadoutMap_ = pSetup->getHcalMapping();
 
   // get the ecal electronics map
-  edm::ESHandle<EcalElectronicsMapping> ecalmapping;
-  iSetup.get<EcalMappingRcd>().get(ecalmapping);
+  const auto ecalmapping = iSetup.getHandle(ecalMappingToken_);
```

Comment:
```suggestion
  EcalMapping_ = iSetup.getData(ecalMappingToken_);
```

Reply:
`getData()` returns a const & but the member variable is a pointer.